src/v2/ion/i18nTransformer.js

```diff
@@ -288,13 +332,22 @@ const isWebAuthnAPIError = ( i18nKey ) => i18nKey.startsWith(WEBAUTHN_API_GENERI
  */
 const getMessage = (message) => {
   if (message.i18n?.key) {
-    const i18nKey = I18N_OVERRIDE_MAPPINGS[message.i18n?.key] ?? message.i18n.key;
+    let i18nKey = message.i18n.key;
+    let i18nParms = message.i18n.params || [];
+
+    if (I18N_OVERRIDE_MAPPINGS[message.i18n?.key]) {
+      i18nKey = I18N_OVERRIDE_MAPPINGS[message.i18n?.key];
+    } else if (I18N_OVERRIDE_WITH_PARAMS_MAP[i18nKey]) {
+      const param = message.i18n.params?.[0];
+      i18nKey = getI8nKeyUsingParams(i18nKey, param);
+      i18nParms = [];
```

Comment:
nit: `i18nParams`


Can you share an example of what the response look like for `registration.error.invalidLoginEmail.custom` ? I see that the value for this key is `'{0}' must be in the form of an email address` but we set `i18nParms` to []. How does the value for `{0}` get applied in this case?

Reply:
@nikhilvenkatraman-okta good catch, I pushed the commit after your reviewed https://github.com/okta/okta-signin-widget/pull/1912/commits/9968de5cd2efe20d0b60c9b0f4265d5ca9f09445#diff-c64539a9635d1e00f39eaf1a3c3e8b634aed28ae450ad9a1740fefd6076b06dbR346

Reply:
Documenting the custom case: https://github.com/okta/okta-signin-widget/pull/1912/commits/3798d028dc6397d9474761b51a95377a36abfa28

Also see PR description for screenshots